transforms/universal/filter/ray/src/filter_transform_ray.py

```diff
@@ -10,203 +10,17 @@
 # limitations under the License.
 ################################################################################
 
-import argparse
-import ast
-import json
-
-import duckdb
-import pyarrow as pa
-from data_processing.runtime.pure_python.runtime_configuration import (
-    PythonTransformRuntimeConfiguration,
-)
-from data_processing.transform import AbstractTableTransform, TransformConfiguration
-from data_processing.utils import CLIArgumentProvider, get_logger
+from data_processing.utils import get_logger
 from data_processing_ray.runtime.ray import RayTransformLauncher
 from data_processing_ray.runtime.ray.runtime_configuration import (
     RayTransformRuntimeConfiguration,
 )
+from filter_transform import FilterTransformConfiguration
```

Comment:
Oh, its a monster, that is based on a clever make files, but requires a lot of pycharm knowledge and very hard to understand and reason